ktext.md

```diff
@@ -353,25 +353,29 @@ To figure out what kind of keyboard and trackpad you have, check Device Manager
 
 * [VoodooPS2](https://github.com/acidanthera/VoodooPS2/releases)
   * For systems with PS2 keyboards, mice, and trackpads
-  * Requires OS X 10.11 or newer for MT2 (Magic Trackpad 2) functions
+  * Requires macOS 10.11 or newer for MT2 (Magic Trackpad 2) functions
```

Comment:
Any specific reason to not leave the legacy naming?

Reply:
macOS is currently how Apple brands the OS, and it's a bit clearer than "OS X", additional OS X is a bit redundant since it's Operating System 10 10.11